lib/OrdinaryDiffEqLowStorageRK/src/alg_utils.jl

```diff
@@ -0,0 +1,131 @@
+alg_order(alg::CKLLSRK54_3C_3R) = 4
+alg_order(alg::CKLLSRK54_3M_3R) = 4
+alg_order(alg::CKLLSRK54_3N_3R) = 4
+alg_order(alg::CKLLSRK85_4C_3R) = 5
+alg_order(alg::CKLLSRK85_4M_3R) = 5
+alg_order(alg::RDPK3Sp35) = 3
+alg_order(alg::RDPK3SpFSAL35) = 3
+alg_order(alg::RDPK3Sp49) = 4
+alg_order(alg::RDPK3SpFSAL49) = 4
+alg_order(alg::RDPK3Sp510) = 5
+alg_order(alg::RDPK3SpFSAL510) = 5
+alg_order(alg::CKLLSRK85_4P_3R) = 5
+alg_order(alg::CKLLSRK85_4FM_4R) = 5
+alg_order(alg::CKLLSRK54_3N_4R) = 4
+alg_order(alg::CKLLSRK75_4M_5R) = 5
+alg_order(alg::CKLLSRK54_3M_4R) = 4
+alg_order(alg::CKLLSRK65_4M_4R) = 5
+
+isfsal(alg::ORK256) = false
+isfsal(alg::CarpenterKennedy2N54) = false
+isfsal(alg::DGLDDRK84_F) = false
+isfsal(alg::DGLDDRK73_C) = false
+isfsal(alg::DGLDDRK84_C) = false
+isfsal(alg::RDPK3Sp35) = false
+isfsal(alg::RDPK3Sp49) = false
+isfsal(alg::RDPK3Sp510) = false
+isfsal(alg::SHLDDRK64) = false
+isfsal(alg::NDBLSRK134) = false
+isfsal(alg::NDBLSRK124) = false
+isfsal(alg::NDBLSRK144) = false
+
+uses_uprev(alg::ORK256, adaptive::Bool) = false
+uses_uprev(alg::SHLDDRK64, adaptive::Bool) = false
+uses_uprev(alg::CarpenterKennedy2N54, adaptive::Bool) = false
+uses_uprev(alg::NDBLSRK124, adaptive::Bool) = false
+uses_uprev(alg::NDBLSRK134, adaptive::Bool) = false
+uses_uprev(alg::DGLDDRK84_F, adaptive::Bool) = false
+uses_uprev(alg::NDBLSRK144, adaptive::Bool) = false
+uses_uprev(alg::DGLDDRK84_C, adaptive::Bool) = false
+uses_uprev(alg::TSLDDRK74, adaptive::Bool) = false
+uses_uprev(alg::CFRLDDRK64, adaptive::Bool) = false
+uses_uprev(alg::DGLDDRK73_C, adaptive::Bool) = false
+uses_uprev(alg::CKLLSRK43_2, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3C, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK95_4S, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK95_4C, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK95_4M, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3C_3R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3M_3R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3N_3R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK85_4C_3R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK85_4M_3R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK85_4P_3R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3N_4R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK54_3M_4R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK65_4M_4R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK85_4FM_4R, adaptive::Bool) = adaptive
+uses_uprev(alg::CKLLSRK75_4M_5R, adaptive::Bool) = adaptive
+
+"""
+    ssp_coefficient(alg)
+
+Return the SSP coefficient of the ODE algorithm `alg`. If one time step of size
+`dt` with `alg` can be written as a convex combination of explicit Euler steps
+with step sizes `cᵢ * dt`, the SSP coefficient is the minimal value of `1/cᵢ`.
+
+# Examples
+
+```julia-repl
+julia> ssp_coefficient(SSPRK104())
+6
+```
+"""
+
+ssp_coefficient(alg::KYK2014DGSSPRK_3S2) = 0.8417
```

Comment:
```suggestion
"""
ssp_coefficient(alg::KYK2014DGSSPRK_3S2) = 0.8417
```

Otherwise, the docstring is not attached to the method